### PR TITLE
gomod: Update dependencies

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

**Review by breml — APPROVED** [review]
LGTM

File: internal/instance/resource_instance_test.go [hunk 1]
```diff
@@ -556,7 +556,10 @@ func TestAccInstance_fileUploadContent_VM(t *testing.T) {
 	instanceName := petname.Generate(2, "-")
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { acctest.PreCheck(t) },
+		PreCheck: func() {
```
[the remark this answers is not in the document]
Comment (stgraber) [reply internal/instance/resource_instance_test.go:559]: Yes, but those tests already have the correct PreCheck, the precheck is just missing the github actions detection part.

[the remark this answers is not in the document]
Comment (maveonair) [reply internal/instance/resource_instance_test.go:559]: Why are these tests suddenly starting to fail?

[the remark this answers is not in the document]
Comment (stgraber) [reply internal/instance/resource_instance_test.go:559]: Github changed the type of VMs they're using to run Github Actions.

**Review by maveonair — APPROVED** [review]

**Pull request merged**